frontend/webpack.circular-deps.ts

```diff
@@ -109,7 +123,12 @@ export class CircularDependencyPreset {
       new CircularDependencyPlugin({
         exclude: this.options.exclude,
         onDetected: ({ module: { resource }, paths: modulePaths }) => {
-          cycles.push({ causedBy: resource, modulePaths });
+          if (
+            typeof this.options.filterModules === 'undefined' ||
```

Comment:
Does this miss `null`? Maybe

```suggestion
            !this.options.filterModules ||
```

Reply:
Committed! Thanks for the catch